thrift/shared.thrift

```diff
@@ -1797,11 +1797,24 @@ struct CrossClusterSignalExecutionRequestAttributes {
 struct CrossClusterSignalExecutionResponseAttributes {
 }
 
+struct CrossClusterRecordChildWorkflowExecutionCompleteRequestAttributes {
+  10: optional string targetDomainID
+  20: optional string targetWorkflowID
+  30: optional string targetRunID
+  40: optional i64 (js.type = "Long") initiatedEventID
+  50: optional WorkflowExecution completedExecution
```

Comment:
Is this for the source (child) workflow? If so, that information is already available in `CrossClusterTaskInfo`. Please check L1813 below and L1749.